src/client/datascience/interactive-window/interactiveWindowCommandListener.ts

```diff
@@ -341,6 +341,7 @@ export class NativeInteractiveWindowCommandListener implements IDataScienceComma
             // Create a new notebook
             notebook = await this.notebookProvider.getOrCreateNotebook({
                 identity: createExportInteractiveIdentity(),
+                disableUI: false,
```

Comment:
If we're exporting, then notebooks are started, the UI should be displayed. Its not a hidden action, the user chose to do this.